product-service/src/handlers/catalog-batch-process.ts

```diff
@@ -0,0 +1,51 @@
+import AWS from 'aws-sdk';
+import { APIGatewayProxyHandler } from 'aws-lambda';
+
+const { SQS_URL, SQS_REGION } = process.env;
+const DEFAULT_HEADERS = {
+  'Access-Control-Allow-Origin': '*'
+};
+
+const sendMessages = async ({ messages, sqs, queueUrl }) =>
```

Comment:
Product service is required to read from SQS, not to send messages. At least not to the same queue.

Reply:
My bad, updated, thanks